.gitignore

```diff
@@ -44,3 +44,4 @@ ObjectStore
 *.hprof
 /.nb-gradle/
 
+*.plantuml
```

Comment:
Just out of curiosity, what is this for?

Reply:
Ooops, didn't mean to commit that. Admittedly its harmless - its for ignoring the diagrams generated by https://plugins.jetbrains.com/plugin/10387-sketch-it-/